testsuite/tests/lib-list/test.ml

```diff
@@ -11,6 +11,9 @@ let string_of_even_opt x =
 let () =
   let l = List.init 10 (fun x -> x) in
   let sl = List.init 10 string_of_int in
+  assert (List.iota 3 = [0; 1; 2]);
+  assert (List.iota 5 ~start:1 = [1; 2; 3; 4; 5]);
+  assert (List.iota 5 ~start:10 ~step:2 = [20; 22; 24; 26; 28]);
```

Comment:
shouldn't this, based on the description in the mli, result in `[ 10 ; 12 ; 14 ; 16 ; 18 ]`?

Reply:
Embarrassingly enough you're right; I'm sending a fix soon.